src/Displaz.jl

```diff
@@ -208,6 +208,14 @@ function newfigure()
     figure(id)
 end
 
+struct convert_values{T} end
+
+convert_values{:classification}(v)  = (v isa Integer) ? UInt8(v) : error("classification must be integer.")
```

Comment:
Instead of making a `convert_values(c::T) where T <: Integer` kind of construction, I want to allow all types and use `isa Integer`, to, A, throw a nicer error than "MethodError", and, B, throw an error to begin with, else it would fall back to the generic `convert_values`.